src/test/java/core/basesyntax/validation/DataValidatorTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.validation;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class DataValidatorTest {
+    private final DataValidator dataValidator = new DataValidator();
+
+    @Test
+    public void checkForEmptyList_validate_Ok() {
+        List<String> inputData = Arrays.asList("type,fruit,quantity");
```

Comment:
The test method name does not reflect the actual test being performed. It is named `checkForEmptyList_validate_Ok` but it does not check for an empty list. Consider renaming it to something more accurate, like `correctHeader_validate_Ok`.